cucumber/config/protractor-parallel.config.js

```diff
@@ -0,0 +1,42 @@
+"use strict";
+
+const GLOBAL_TIMEOUT = 40e3;
+
+exports.config = {
+  specs: "../features/*.feature",
+  multiCapabilities: [
+    { browserName: "chrome", shardTestFiles: true, maxInstances: 1 },
+    {
+      browserName: "firefox",
```

Comment:
Could you please delete this browser and add another chrome?

Reply:
done